virttest/virt_vm.py

```diff
@@ -1242,7 +1245,10 @@ def serial_login(self, timeout=LOGIN_TIMEOUT,
         console.set_status_test_command(status_test_command)
 
         remote.handle_prompts(console, username, password, prompt, timeout)
```

Comment:
IMHO, the `create_console()` method should not be called any more in this function, and all the setup steps (including `console.set_linesep`, `console.set_status_test_command`, `remote.handle_prompts` ...) should be wrapped in `ConsoleManager`.